pkg/metrics/metrics.go

```diff
@@ -0,0 +1,104 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	OCMAgentTotalRequests = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "ocm_agent_total_requests",
```

Comment:
https://prometheus.io/docs/practices/naming/ we should follow prometheus conventions?

```suggestion
			Name: "ocm_agent_requests_total",
```

Reply:
+1